NEWS.md

```diff
@@ -243,8 +244,8 @@ Deprecated or removed
 [#15172]: https://github.com/JuliaLang/julia/issues/15172
 [#15192]: https://github.com/JuliaLang/julia/issues/15192
 [#15242]: https://github.com/JuliaLang/julia/issues/15242
+[#15244]: https://github.com/JuliaLang/julia/issues/15244
 [#15258]: https://github.com/JuliaLang/julia/issues/15258
-[#15409]: https://github.com/JuliaLang/julia/issues/15409
```

Comment:
The mention of the commit was removed by @amitmurthy in 52f9cb63.

(ps `git log -p NEWS.md`, and then search with `/` for `15409`)